### DISCUSSION TITLE
Keycloak service

### DISCUSSION THREAD

**Discussion by rodgons** [coollabsio/coolify#1819] [Service Template Requests]
Hi, I’m new to project and I’d seen that coolify had a keycloak service but I can’t find it now.
Was it removed? If so, does anyone can give me a help on how to deploy it on my own?
I’ve read the doc on how to create a service but I couldn’t do it properly

**Comment by peaklabs-dev** [comment]
@rodgons Will be added soon: https://github.com/coollabsio/coolify/pull/3109

**Comment by peaklabs-dev** [comment]
This has been implemented and will be available in an upcoming release. Thank you for your patience.
